docs/myst.yml

```diff
@@ -9,6 +9,35 @@ project:
   references:
     python: https://docs.python.org/3.7/
     jupyterbook: https://jupyterbook.org/en/stable/
+  abbreviations:
+    AST: Abstract Syntax Tree
+    DOI: Digital Object Identifier
+    MyST: Markedly Structured Text
+    CLI: Command Line Interface
```

Comment:
@choldgraf @mmcky -- I am pretty jazzed about this feature. It is an extremely easy way to add abbreviations to a myst project. You add them to the frontmatter, and then it replaces them throughout the entire site.

![abbr](https://github.com/executablebooks/mystjs/assets/913249/d4335c25-f84c-4c43-8289-1817a5994d9d)

Reply:
nice, this is really neat